src/pydp/__init__.py

```diff
@@ -6,6 +6,7 @@
 from pydp import distributions
 from pydp import util
 from pydp import ml
+from pydp import partition_selection
```

Comment:
Rather than creating it as the main module, we could move it inside algorithm as a submodule. 
Thoughts @dvadym?

Reply:
Moved to `algorithms`.